src/_posts/platform/internals/stacks/2000-01-01-stacks.md

```diff
@@ -12,12 +12,14 @@ well-known Linux distribution.
 Scalingo currently supports two stacks based on Ubuntu. The corresponding base
 Docker images are open source and can be found on the Docker hub:
 
-{: .table }
+<div class="overflow-horizontal-content" markdown="1">
 | Name | Base Distribution | Supported Until | Docker Hub |
 | ---- | ----------------- | --------------- | ---------- |
 | [scalingo-20]({% post_url platform/internals/stacks/2000-01-01-scalingo-20-stack %}) | Ubuntu 20.04 | April 2025 | [scalingo/scalingo-20](https://hub.docker.com/r/scalingo/scalingo-20) |
 | [scalingo-18]({% post_url platform/internals/stacks/2000-01-01-scalingo-18-stack %}) | Ubuntu 18.04 | April 2023 | [scalingo/scalingo-18](https://hub.docker.com/r/scalingo/scalingo-18) |
 | [scalingo-14]({% post_url platform/internals/stacks/2000-01-01-scalingo-14-stack %}) | Ubuntu 14.04 | December 2019 | [scalingo/scalingo-14](https://hub.docker.com/r/scalingo/scalingo-14) |
+{: .table }
```

Comment:
Why does this move _after_ the Markdown table?

Reply:
Yes @guillaume-sc , I don't know why but it wasn't working if leaving it up: the "implicit" html under markdown wasn't interpreted so it was just plain text.